packages/grid/test/column.test.js

```diff
@@ -555,3 +555,27 @@ describe('column - simple grid', () => {
     expect(getBodyCellContent(grid, 1, 0).textContent.trim()).to.equal('foo1');
   });
 });
+
+describe('column - simple grid with rowHeader', () => {
+  let grid, column;
+
+  beforeEach(async () => {
+    grid = fixtureSync(`
+      <vaadin-grid>
+        <vaadin-grid-column row-header path="value"></vaadin-grid-column>
+      </vaadin-grid>
+    `);
+    column = grid.querySelector('vaadin-grid-column');
+    grid.size = 1;
+    grid.dataProvider = infiniteDataProvider;
+    await nextFrame();
+  });
+
+  it('should look properly in the shadow dom (role=rowheader)', async () => {
+    await nextFrame();
+    const firstCell = grid.shadowRoot.querySelector('tbody tr:first-child td:first-child');
```

Comment:
Why do we need this test? It seems to cover the same logic as the tests in `accessibility.test.js` file.

Reply:
Yes, it is true. I just wanted more coverage (maybe duplicated coverage). And with @tomivirkki on the previous [PR](https://github.com/vaadin/web-components/pull/5843) we removed many test-related changes from `column.test.js` and discussed that this new test case the only thing that can stay in `column.test.js`. 
If you would like/be convinced that I can remove it from here, I have no hard opinion to make it stay here.

Reply:
I think this test can be removed, especially now when the logic is much less complicated.

Reply:
Removed and updated tests in `accessibility.test.js`.